src/sentry/templates/sentry/emails/setup_2fa.txt

```diff
@@ -0,0 +1,11 @@
+Setup Two-Factor Authentication
+
+    Hello {{ user.name }},
+
+    The {{ organization.name }} organization now requires that all members enable
+    two-factor authentication. Effective immediately, you will be unable to access
+    projects under the {{ organization.name }} organization unless you enable at
+    least one form of two-factor authentication.
+
+    Click the link below to enable two-factor authentication.
```

Comment:
For plain text, there is no "click".

Reply:
Also not sure why the entire thing is padded out 4 spaces? Do we have a precedent for this somewhere? I'd just drop that.